WalletWasabi.Gui/ViewModels/StatusBarViewModel.cs

```diff
@@ -319,14 +315,6 @@ private void OnResponseArrivedIsGenSocksServFail(bool isGenSocksServFail)
 					{
 						MainWindowViewModel.Instance.ModalDialog.Close(true);
 					}
-					else
-					{
-						// Do nothing.
-					}
-				}
-				else
-				{
-					// Do nothing.
 				}
 			}
 		}
```

Comment:
Please revert this, too. Following the logic is important here. I remember writing this code and I wrote this code based on the comments. If we find some other exception here we will be able to use the same mental models.